package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "prettier": "^2.8.1",
```

Comment:
This is to format the code automatically. You can setup your IDE so it runs every time you save changes to a file.